cli/tests/stake.rs

```diff
@@ -1218,17 +1264,29 @@ fn test_stake_set_lockup() {
     config_offline.command = CliCommand::ClusterVersion;
     process_command(&config_offline).unwrap_err();
 
-    request_and_confirm_airdrop(&rpc_client, &config, &config.signers[0].pubkey(), 5_000_000)
-        .unwrap();
-    check_balance!(5_000_000, &rpc_client, &config.signers[0].pubkey());
+    request_and_confirm_airdrop(
+        &rpc_client,
+        &config,
+        &config.signers[0].pubkey(),
+        5_000_000_000_000,
+    )
+    .unwrap();
+    check_balance!(5_000_000_000_000, &rpc_client, &config.signers[0].pubkey());
 
-    request_and_confirm_airdrop(&rpc_client, &config_offline, &offline_pubkey, 1_000_000).unwrap();
-    check_balance!(1_000_000, &rpc_client, &offline_pubkey);
+    request_and_confirm_airdrop(
+        &rpc_client,
+        &config_offline,
+        &offline_pubkey,
+        1_000_000_000_000,
+    )
+    .unwrap();
+    check_balance!(1_000_000_000_000, &rpc_client, &offline_pubkey);
 
     // Create stake account, identity is authority
     let minimum_stake_balance = rpc_client
         .get_minimum_balance_for_rent_exemption(StakeState::size_of())
-        .unwrap();
+        .unwrap()
+        + 10_000_000_000;
```

Comment:
nittiest of nits: perhaps this variable name should get changed since it isn't really the minimum stake balance

Reply:
Sounds good!

Reply:
Update: It's `stake_balance` now 😁